packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -79,19 +81,31 @@ export default function ClassifierContainer({
   const allowedWorkflows = canPreviewWorkflows ? project?.links.workflows : project?.links.active_workflows
   const allowedWorkflowID = allowedWorkflows.includes(workflowID) ? workflowID : null
 
+  /** Fetch the workflow object by id using SWR */
   const workflowSnapshot = useWorkflowSnapshot(allowedWorkflowID)
+
+  /** Fetch workflow task strings using SWR. Locale is passed from app-project */
   const workflowTranslation = usePanoptesTranslations({
     translated_id: workflowID,
     translated_type: 'workflow',
     language: locale
   })
+
+  /** Re-assign the fetched translations to the workflow snapshot object */
   if (workflowSnapshot && workflowTranslation) {
     workflowSnapshot.strings = workflowTranslation.strings
   }
 
+  /** Init a mobx store if store is null, or load from session storage when cachePanoptesData is true
+      - storeEnvironment is the auth env and clients
+      - cachePanoptesData is true only for workflow.prioritized
+      - fem-classifier-id is a key
+      - When any of those three variables passed to useHydratedStore update, the useMemo in useHydratedStore runs
+   */
   const classifierStore = useHydratedStore(storeEnvironment, cachePanoptesData, `fem-classifier-${project.id}`)
   const { classifications, subjects, userProjectPreferences } = classifierStore
 
+  /** Make sure the classifierStore's active project is in sync with app-project */
   if (project?.id) {
```

Comment:
@eatyourgreens In what UX scenario is this check needed?

Reply:
Dev classifier, I think.

Reply:
Yeah, this will be undefined while the project is loading in the dev classifier.

Reply:
Another scenario for this would be the standalone classifier running in a museum display. We have done this before, but with PFE. The Etch-A-Cell classifier ran on a touchtable at the Crick Institute for several months. That was one of the motivations for building a standalone classifier here.